src/DataLayer/AuxBmsData/AuxBmsData.cpp

```diff
@@ -47,14 +64,89 @@ bool AuxBmsData::getAllowCharge() const
     return allowCharge_;
 }
 
-bool AuxBmsData::getContactorError() const
+bool AuxBmsData::getHighVoltageEnableState() const
+{
+    return highVoltageEnableState_;
+}
+
+bool AuxBmsData::getAllowDischarge() const
+{
+    return allowDischarge_;
+}
+
+bool AuxBmsData::getOrionCANReceivedRecently() const
+{
+    return auxMaskedBit(ORION_CAN_RECEIVED_RECENTLY_MASK);
```

Comment:
You have `auxMaskedBit` called with `ORION_CAN_RECEIVED_RECENTLY_MASK` and `CHARGE_TRIP_DUE_TO_HIGH_CELL_VOLTAGE_MASK`, which are both `0x01`. Did you mean to create a `auxContactorMaskedBit` function too?